content/docs/use-cases/ci-cd-for-machine-learning.md

```diff
@@ -0,0 +1,89 @@
+# Continuous Integration and Deployment for Machine Learning
+
+Applying DevOps methodologies to machine learning (MLOps) and data management
+(DataOps) is increasingly common. This means resource orchestration
+(provisioning servers), testing, and deployment to production (CI/CD --
+continuous integration and continuous delivery), as well as monitoring &
+feedback.
+
+![](https://static.iterative.ai/img/ml-pipeline.png) _Basic ML pipeline_
+
+The main benefits of CI/CD for ML are detailed below.
+
+### Automating Quality Assurance (QA)
```

Comment:
Wouldn't it be better to use more ML/data specific cases like:
1. [Model environment testing] Make sure the system can reproduce a proper environment (packages, libs) and make an inference.
2. [ML model validating] Make sure model scores are within a certain limitation/threshold.
3. [ML training in clouds] ...
4. [Data quality control] Validate data sources: no empty files, variables are within a specified standard deviation etc.

Reply:
integrated these ideas & simplified (removed) headings